recipes/boost/all/conanfile.py

```diff
@@ -55,7 +55,8 @@ class BoostConan(ConanFile):
         "segmented_stacks": [True, False],
         "debug_level": [i for i in range(1, 14)],
         "pch": [True, False],
-        "extra_b2_flags": "ANY"  # custom b2 flags
+        "extra_b2_flags": "ANY",  # custom b2 flags
+        "icu": [True, False],
```

Comment:
are ICU and iconv mutually exclusive options?
if yes, maybe it's better to design option like `locale_backend: ["ICU", "iconv", None]`
this might be helpful: https://github.com/bincrafters/conan-boost_base/blob/testing/2.0.0/conanfile.py
/cc @grafikrobot

Reply:
I think that `iconv` is more like a fallback, you can choose to use `icu`, but if it is not available, the package will use `iconv`. It also follows the same option pattern as `zlib`, `zstd`,... if  you activate it then more requires will be added.

Reply:
they are different. zlib, zstd, bzip2 backends can be used together, while iconv and ICU seems to be mutually exclusive.